backbone.js

```diff
@@ -644,7 +646,7 @@
       if (attrs && wait) this.attributes = _.extend({}, attributes, attrs);
 
       var method = this.isNew() ? 'create' : (options.patch ? 'patch' : 'update');
-      if (method === 'patch' && !options.attrs) options.attrs = attrs;
+      if (method === 'patch' && !options.attrs) options.attrs = this.changedAttributes(attrs);
```

Comment:
You can override `Model#save` or `Model#sync` to add this in your own app if you need it.